client/src/controllers/DropdownController.ts

```diff
@@ -25,8 +25,24 @@ export class DropdownController extends Controller<HTMLElement> {
   declare readonly toggleTarget: HTMLButtonElement;
   declare readonly contentTarget: HTMLDivElement;
   declare readonly hideOnClickValue: boolean;
+  tippy?: Instance<Props>;
 
   connect() {
+    this.tippy = tippy(this.toggleTarget, this.options);
+  }
+
+  hide() {
+    this.tippy?.hide();
+  }
+
+  show() {
+    this.tippy?.show();
+  }
+
+  /**
+   * Default Tippy Options
+   */
+  get options(): Partial<Props> {
```

Comment:
I noticed an error that surfaced as a result of #10557:

The following line in `connect()` (now in this `options()` property):

```ts
this.contentTarget.hidden = false;
```

would trigger an error for controllers that do not have a `content` target. This can happen if a controller is disconnected and reconnected again, because tippy would replace the content target with its own tippy-controlled element.

An instance of this issue can be triggered by using the custom ordering in the page explorer view. After you drag-and-drop a page, the controller will disconnect and fail to reconnect (an error is shown on the console), but the dropdown will still work because tippy seems to still have control over it.

I think a simple solution would be to check for `this.hasContentTarget` and do an early return if it's `false`.

Reply:
Hmm. Good find. Have you tested that scenario on this branch?

This approach adds a disconnect method to destroy any tippy instance also.

Reply:
@laymonage  good find, I have done some digging and have made the suggested changes plus also removed the `destroy` callback as it is not suitable for the way the dropdown content gets moved by Tippy.

Unlike the simpler tooltip variants it cannot 'rebuild' from the DOM as is once Tippy has set up the content.